lib/mayaUsd/ufe/UsdUndoCreateGroupCommand.cpp

```diff
@@ -83,6 +89,19 @@ void UsdUndoCreateGroupCommand::execute()
             append(parentCmd);
         }
     }
+
+    // Make sure to add the newly created _group (a.k.a parent) to selection. This matches native
+    // Maya behavior and also prevents the crash on grouping a prim twice.
+    auto noWorld = _group->path().popHead();
+    auto fullPath = noWorld.getSegments().front().string() + Ufe::PathString::pathSegmentSeparator()
+        + noWorld.getSegments().back().string();
+
+    MString groupPath(fullPath.c_str());
+    MGlobal::executeCommand(MString("select -r \"") + groupPath + "\" ");
+
+    TF_VERIFY(
+        Ufe::GlobalSelection::get()->size() == 1,
+        "_group node should be in the global selection now. \n");
```

Comment:
We previously had nothing selected after group was created. This resulted in crash in UsdUndoInsertChildCommand if one tries to call the group the second time without having something selected.

Reply:
@ppt-adsk  Thinking more about this... Made the logic even simpler:

37e51f1857dc69bc938c8b41d445f5380fcdf4f4